java/src/test/java/com/pryv/acceptance/ConnectionAccessesTest.java

```diff
@@ -0,0 +1,230 @@
+package com.pryv.acceptance;
+
+import com.jayway.awaitility.Awaitility;
+import com.pryv.Connection;
+import com.pryv.Filter;
+import com.pryv.database.DBinitCallback;
+import com.pryv.interfaces.CreateCallback;
+import com.pryv.interfaces.DeleteCallback;
+import com.pryv.interfaces.GetCallback;
+import com.pryv.interfaces.UpdateCallback;
+import com.pryv.model.Access;
+import com.pryv.model.ApiResource;
+import com.pryv.model.Permission;
+import com.pryv.utils.Logger;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import resources.TestCredentials;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class ConnectionAccessesTest {
```

Comment:
In general, your test cases work because they are run alphabetically, can you either make them independent or specify the order? Because they will break if you change the test methods' names.

Reply:
Yes thanks

Reply:
I looked at this in more detail and I think there is no problem with the order. I intentionally put the creation test in @BeforeClass and the deletion test in @AfterClass. Then, the get and update test can be executed in any order.

Reply:
So it is more an issue of form, because the `beforeclass` is technically not part of the tests, neither is `afterclass`